src/index.mjs

```diff
@@ -113,6 +114,20 @@ export class Circuit extends HeadlessCircuit {
     displayOn(elem) {
         return this._makePaper(elem, this._graph);
     }
+    zoomIn(paper) {
+        this.currentZoomIndex++;
+        this._scaleAndRefreshPaper(paper);
+    }
+    zoomOut(paper) {
+        this.currentZoomIndex--;
+        this._scaleAndRefreshPaper(paper);
+    }
+    _scaleAndRefreshPaper(paper, scale = this.currentZoomIndex, graph) {
+        paper.scale(Math.pow(1.1, scale));
+        if (graph) {
+            graph.resetCells(graph.getCells());
```

Comment:
Is this operation really needed? And if it is, why no freeze/unfreeze of the paper? This could have performance implications.

Reply:
I fixed it, and I freeze, and unfreeze the graph now.